.github/workflows/cleanup.yaml

```diff
@@ -0,0 +1,36 @@
+name: Cleanup
+
+on:
+  pull_request:
+    types: [closed, unlabeled]
+    branches:
+      - maintenance-3.0.x
+
+env:
+  GKE_CLUSTER: jx-prod
+  GKE_ZONE: us-east1-b
+
+jobs:
+  cleanup:
+    name: Preview
+    if: (github.event.action == 'unlabeled' && github.event.label.name == 'preview') || contains(github.event.pull_request.labels.*.name, 'preview')
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Prepare environment
+      run: |
+        BRANCH_NAME=${GITHUB_HEAD_REF##*/}
+        echo "PREVIEW_NAMESPACE=nuxeo-elements-${BRANCH_NAME,,}" >> $GITHUB_ENV
```

Comment:
nit: are all these commas necessary?

Reply:
yep, it's a lowercase parameter expansion operator, see https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#Shell-Parameter-Expansion
but this has to be updated according to changes done to the preview namespace anyway. 
Will fix it 👍